browser/ui/views/pulsing_block/pulsing_block_view.h

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_BROWSER_UI_VIEWS_PULSING_BLOCK_PULSING_BLOCK_VIEW_H_
+#define BRAVE_BROWSER_UI_VIEWS_PULSING_BLOCK_PULSING_BLOCK_VIEW_H_
+
+#include <memory>
+
+#include "base/compiler_specific.h"
+#include "base/timer/timer.h"
+#include "brave/browser/ui/views/pulsing_block/tooltip_webui_bubble_view.h"
+#include "chrome/browser/ui/browser.h"
+#include "ui/views/view.h"
+
+namespace gfx {
+class Size;
+}
+
+namespace views {
```

Comment:
Acutally, //brave/browser//ui/views shouldn't have `namespace views`. It's only for classes in //ui/views